.github/workflows/continuous-build.yml

```diff
@@ -122,3 +122,16 @@ jobs:
             public.ecr.aws/xray/aws-xray-daemon:alpha
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache
+
+      - name: Verify alpha image from ECR
+        if: ${{ runner.os == 'Linux' }}
+        run: |
+          docker pull public.ecr.aws/xray/aws-xray-daemon:alpha
```

Comment:
Do we need to use docker for this? A test is better if it can prevent uploading a bad build even for alpha. I think docker has no relation to input validation so we can test with the daemon binary before pushing.

Reply:
Move test forward to build binary